rugby.ipynb

```diff
@@ -0,0 +1,3579 @@
+{
```

Comment:
<p>prior  should not be an extra variable here. To sample from the prior, sample_prior_predictive  must be used, following the same pattern as in the eight schools examples. This automatically generates a dictionary with all the variables in the model: home, sd_att, sd_def, ..., home_points, away_points that is used to populate prior and prior predictive groups (ArviZ knows how to split the dict into the two groups). As I commented before, <code>pm.Flat</code>  is incompatible with prior sampling. Therefore, all instances of it have to be removed and replaced by something compatible with prior sampling. This means modifiying two lines and deleting the <code>prior = pm.Norma...</code>  one:</p><p> </p><pre class="ql-syntax" spellcheck="false">home = pm.Normal('home', mu=0, sigma=5)
intercept = pm.Normal('intercept', mu=0, sigma=5)
</pre><p> </p><p>More on this below.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz_example_data/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thank you for the detailed explanation. I have made the required changes in the file.</p>